featuremanagement/_models/_targeting_context.py

```diff
@@ -0,0 +1,19 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
```

Comment:
Is the convention of this repo to add an emtpy line between copyright and imports?

Reply:
The same issue also occurs in other files.

Reply:
We are using the Python https://azure.github.io/azure-sdk/python_design.html, which includes using the black formatter and pylint. None of these specifies a style for this.